opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -650,8 +650,8 @@ class WellModelMatrixAdapter : public Dune::AssembledLinearOperator<M,X,Y>
         void makeOverlapRowsInvalid(Matrix& ebosJacIgnoreOverlap) const
         {
             //value to set on diagonal
-            typedef Dune::FieldMatrix<Scalar, numEq, numEq >        MatrixBlockType;
-            MatrixBlockType diag_block(0.0);
```

Comment:
why this one?

Reply:
I guess it shadows some types somewhere.

Reply:
It is good if it still has the `Type` in it to say it is a type name.

Reply:
Yes it shadows. Since the typedef is not reused, i removed it.